script.js

```diff
@@ -1,4 +1,193 @@
+var NUMBER_OF_DICE_PLAYED = 2;
+
+var STATE_DICE_ROLL = "DICE_ROLL";
+var STATE_DICE_ORDER = "DICE_ORDER";
+var STATE_AUTO_GENERATE_COMBINED_NUMBER = "AUTO_GENERATE_COMBINED_NUMBER";
+
+var MODE_NORMAL = "Normal";
+var MODE_LOWEST_WINS = "Lowest Wins";
+
+//current default - only 2 players
+var playerOne = [];
+var playerOneFinal = 0;
+var playerOneScore = 0;
+var playerTwo = [];
+var playerTwoFinal = 0;
+var playerTwoScore = 0;
+
+var currentPlayer = 1;
+var currentState = STATE_DICE_ROLL;
+var currentMode = MODE_NORMAL;
+
+var getDiceRolled = function () {
+  //get a random dice roll
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+var playerDiceRoll = function (player, rolls) {
+  for (var i = 0; i < rolls; i += 1) {
+    var rolled = getDiceRolled();
+    player.push(rolled);
+  }
+};
+
+var getCurrentPlayer = function () {
+  if (currentPlayer == 1) return playerOne;
+  else return playerTwo;
```

Comment:
The 'else' keyword here is not necessary since you are returning something above.